RPIPs/RPIP-42.md

```diff
@@ -23,18 +23,22 @@ This work is based on prior work; a copy can be found [here](../assets/rpip-42/b
 
 ## Specification
 - The protocol SHALL be able to penalize stake at the node level
-- Node Operators with <`base_mp_num` base minipools SHALL be able to make a base minpool
-  - A base minipool SHALL have a `base_mp_bond` bond from the Node Operator
-- Node Operators with >=`base_mp_num` base minipools SHALL be able to make a satellite minipool
-  - A satellite minipool SHALL have a `satellite_mp_bond` bond from the Node Operator
-- If a Node has satellite minipools active while that Node has <`base_mp_num` base minipools active, funds SHALL NOT be withdrawable by the Node Operator
-  - It is NOT RECOMMENDED to exit base minipools unless all satellite minipools have been exited
-  - A user MAY exit satellite minipools to allow for fund withdrawal 
-  - The protocol MAY or MAY NOT provide a path to upgrade a satellite minipool to a base minipool (to allow for fund withdrawal)
+- Minipool deposits SHALL be disabled
+- Node Operators SHALL NOT be required to stake RPL in order to create validators within their megapool
+- Node Operators with <`base_num` base validators SHALL be able to make a base validator within their megapool
+  - A base validator SHALL have a `base_bond` bond from the Node Operator
+  - If there is a node operator queue, base validators SHALL be given priority over satellite validators
+- Node Operators with >=`base_num` base validators SHALL be able to make a satellite validator
+  - A satellite validator SHALL have a `satellite_bond` bond from the Node Operator
```

Comment:
Suggest defining satellite validator. I don't know what it means and googling did not surface in the RocketPool docs

Reply:
This is the document that defines them. It defines their size, when they can be made etc